client/object_put.go

```diff
@@ -21,11 +21,17 @@ import (
 	"github.com/nspcc-dev/neofs-sdk-go/user"
 )
 
+var (
+	// ErrIgnoreSession is a special error to show auto-session is disabled.
```

Comment:
It's more of a `ErrNoSessionExplicitly`, it's very similar to `ErrNoSession`.

Reply:
Renamed